src/otpauth/types.py

```diff
@@ -0,0 +1,3 @@
+import typing as t
+
+SupportedAlgorithms = t.Literal["SHA1", "SHA256", "SHA512"]
```

Comment:
How about putting this type into `core.py`, so that we don't have to add a new file.

Reply:
Did this at first, but created a new file as I'm using it as well in the other files. I can revert if you prefer.